src/app/rosetta/lib/block.ml

```diff
@@ -556,6 +556,18 @@ module Specific = struct
         else env.db_block query
       in
       let coinbase = (res#genesisConstants)#coinbase in
+      let%map coinbase_receiver =
+        match
+          List.find block_info.internal_info ~f:(fun info ->
+              info.Internal_command_info.kind = `Coinbase )
+        with
+        | Some cmd ->
+            M.return cmd.Internal_command_info.receiver
+        | None ->
+            M.fail
+            @@ Errors.create ~context:"A coinbase is missing from a block"
+                 `Invariant_violation
```

Comment:
Fyi, currently we do allow blocks without coinbase (when there is no snark work available to purchase)